input/fsh/modules/files.fsh

```diff
@@ -139,4 +161,24 @@ Description: "Information about a file related to a research participant"
 * description ^short = "A description of the file"
 * type 0..1 /*File Type*/
 * type from edam-ontology-terms (extensible) 
-* type ^short = "The type of data contained in this file."
\ No newline at end of file
+* type ^short = "The type of data contained in this file."
+
+Profile: NcpiDRSFile
+Parent: NcpiFile
+Id: ncpi-drs-file
+Title: "NCPI DRS File"
+Description: "Information about a DRS file related to a research participant"
+* ^version = "0.0.1"
+* ^status = #draft
+* content ^slicing.discriminator.type = #pattern
+* content ^slicing.discriminator.path = "code"
+* content ^slicing.rules = #openAtEnd
+* content ^slicing.ordered = false

```

Comment:
I believe it would be better if we could state that the DRS Attachment URI will always be the first. So, this should be set to true.